vignettes/functionality-flow.Rmd

```diff
@@ -178,3 +178,82 @@ the function `extract_classification_dates`. For each individual with
 inclusion events, the date of the *second* event is extracted as the
 classification date, i.e., the date the OSDC algorithm defines as the
 diagnosis date.
+
+## Classifying the diabetes type {#classifying-diabetes-type}
+
+The second component of the OSDC algorithm classifies individuals from
+the extracted diabetes population as having either type 1 or type 2
+diabetes. The output of this component is a `data.frame` that includes
+the diabetes type as well as the classification date (i.e., the date
+of the second instance of the inclusion criteria described in the
+sections above).
+
```

Comment:
As we discussed at the steering group meeting, the output should probably contain two versions of the date of diagnosis/classification date:
1: The raw date of diagnosis
2: A stable date of diagnosis, that is basically the raw date mutated so only cases diagnosed in the time-period where data coverage is sufficient to make incident cases reliable (that they are actually new cases, not just because the data to include them wasn't included before).

Reply:
Ah, nice. I have added this information, as well as a link to the vignette you're currently writing #69